local-build.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+rm -f com.github.shonumi.gbe-plus.flatpak
```

Comment:
No need to have this file here.

Reply:
I like it, and have something similar in all my Flathub projects.